app/config/supervisord.conf

```diff
@@ -0,0 +1,42 @@
+[supervisord]
+nodaemon=true
+pidfile=/tmp/supervisord.pid
+logfile=/dev/fd/1
+logfile_maxbytes=0
+
+[program:x11]
+priority=0
+command=/usr/bin/Xtigervnc -desktop "FSL" -rfbport 5900 -SecurityTypes None -AlwaysShared -AcceptKeyEvents -AcceptPointerEvents -AcceptSetDesktopSize -SendCutText -AcceptCutText :0
+autorestart=true
+stdout_logfile=/dev/fd/1
+stdout_logfile_maxbytes=0
+redirect_stderr=true
+
+[program:novnc]
+priority=0
+directory=/usr/share/novnc
+command=/usr/share/novnc/utils/novnc_proxy --vnc localhost:5900 --listen 8080 --web /usr/share/novnc
+autorestart=true
+stdout_logfile=/dev/fd/1
+stdout_logfile_maxbytes=0
+redirect_stderr=true
+
+[program:openbox]
+priority=1
+command=/usr/bin/openbox
+environment=DISPLAY=:0
+autorestart=true
+stdout_logfile=/dev/fd/1
+stdout_logfile_maxbytes=0
+redirect_stderr=true
+
+[program:app]
+priority=1
+environment=DISPLAY=:0
+directory=/home/scu/work
+# TODO: Change according to you needs
+command=fsl
+autorestart=true
+stdout_logfile=/dev/fd/1
+stdout_logfile_maxbytes=0
```

Comment:
minor: Maybe set this to a very high number instead of 0 which probably means unlimited

Reply:
Done (I'll put max vals of 50MB and 150 MB. I might update that in the cookiecutter for UI services as well.